scripts/galvo_phase_gain_calibration.py

```diff
@@ -0,0 +1,335 @@
+            self.imageCtrl = wx.StaticBitmap(self.panel, wx.ID_ANY, wx.BitmapFromImage(self.img))
+            self.imageCtrlSine = wx.StaticBitmap(self.panel, wx.ID_ANY, wx.BitmapFromImage(self.img))
+            self.imageCtrlText = wx.StaticBitmap(self.panel, wx.ID_ANY, wx.BitmapFromImage(self.img))
+        else:
+            self.img = wx.Image(*size, clear=True)
+            self.imageCtrl = wx.StaticBitmap(self.panel, wx.ID_ANY, wx.Bitmap(self.img))
+            self.imageCtrlSine = wx.StaticBitmap(self.panel, wx.ID_ANY, wx.Bitmap(self.img))
+            self.imageCtrlText = wx.StaticBitmap(self.panel, wx.ID_ANY, wx.Bitmap(self.img))
+        self.panel.SetFocus()
+        self.frame.Show()
+
+    def update_view(self):
+        """Update view with image and sine/text overlay when a new image is received."""
+        logging.debug("Received a new image of %d x %d", *self.img.GetSize())
+        self.frame.ClientSize = self.img.GetSize()
+
+        height = self.img.Height
+        width = self.img.Width
+        point_surface = cairo.ImageSurface(cairo.FORMAT_ARGB32, width, height)
+        ctx = cairo.Context(point_surface)
+        ctx.set_source_rgba(0.5, 0.1, 0.1, 0.7)  # r, g, b, alpha
+        ctx.scale(width, height)
+        cols = numpy.arange(min(self.cols), max(self.cols))
+        sin_y = _sine_wave(cols, *self.params) * self.magn / height
+        sin_x = cols * self.magn / width
+        sine = numpy.vstack((sin_x, sin_y)).T
+        point_temp = numpy.array([0, 0])
+        for point in sine:
+            ctx.translate(*point_temp)  # translate back to the origin since point_temp is negative
+            ctx.translate(*point)  # translate from the origin to the coordinate of the point
+            point_temp = numpy.copy(-point)
+            ctx.arc(0, 0, 0.0025, 0, 2 * numpy.pi)
+            ctx.fill()
+        text_surface = cairo.ImageSurface(cairo.FORMAT_ARGB32, width, height)
+        info = [
+            # TODO add voltages when incorporated with AWG's
+            "E-beam scanner: {} V".format("N/A"),
+            "Galvo scanner: {} V".format("N/A"),
+            "Amplitude: {:.2f} um".format(PIXEL_SIZE_UM * numpy.abs(self.params[1])),
+        ]
+        ctx2 = cairo.Context(text_surface)
+        ctx2.set_source_rgb(1.00, 0.83, 0.00)
+        font_size = 20
+        ctx2.set_font_size(font_size)
+        ctx2.translate(font_size, font_size)
+        # Cairo doesn't do multiline text plotting, so loop over the text and show at a lower location.
+        for text in info:
+            ctx2.translate(0, font_size)
+            ctx2.show_text(text)
+            ctx2.stroke()
+
+        if wx.MAJOR_VERSION <= 3:
+            self.imageCtrl.SetBitmap(wx.BitmapFromImage(self.img))
+            self.imageCtrlSine.SetBitmap(wx.lib.wxcairo.BitmapFromImageSurface(point_surface))
+            self.imageCtrlText.SetBitmap(wx.lib.wxcairo.BitmapFromImageSurface(text_surface))
+        else:
```

Comment:
In general, you can drop the support for wxPython 3. With Odemis 2.10, wxPython 4 is the default version, and in Ubuntu 18.04, wxPython 3 is not even available.